os/Kconfig.debug

```diff
@@ -45,6 +45,38 @@ config DEBUG_VERBOSE
 
 comment "Subsystem Debug Options"
 
+config DEBUG_WIFICSI
```

Comment:
Let's add it by Alphabet order to find it easily.

Reply:
Done